pkg/kv/kvserver/client_replica_test.go

```diff
@@ -6250,7 +6250,10 @@ func TestMergeDropsLocksIfLargerThanMax(t *testing.T) {
 
 func TestMergeReplicatesLocks(t *testing.T) {
 	defer leaktest.AfterTest(t)()
-	defer log.Scope(t).Close(t)
+	scope := log.Scope(t)
+	defer scope.Close(t)
+
+	skip.UnderDuress(t, "too slow for testrace")
```

Comment:
I can try to follow up here. We can probably get away with a single-node test for this which will probably help a bit.